fusillade/__init__.py

```diff
@@ -21,3 +22,12 @@
 
 
 '''
+
+directory_name = Config.get_directory_name()
+try:
+    directory = CloudDirectory.from_name(directory_name)
+except FusilladeException:
+    from .clouddirectory import publish_schema, create_directory
+    schema_name = Config.get_schema_name()
+    schema_arn = publish_schema(schema_name, version="0.1")
```

Comment:
can version for schema be moved into the Config + Env Value? unless its not going to change much.

Reply:
adding TODO